packages/ts/react-crud/test/autoform.spec.tsx

```diff
@@ -482,6 +484,84 @@ describe('@hilla/react-crud', () => {
       expect(submitButton.disabled).to.be.true;
     });
 
+    describe('form submit', () => {
+      it('submit when enter key is pressed', async () => {
```

Comment:
```suggestion
      it('submits when enter key is pressed', async () => {
```

Nit: Prefer to keep test cases names consistent with the `it` prefix. Ideally it reads like: "It submits then the enter key is pressed", or "It should submit when the enter key is pressed".